### ISSUE TITLE
Add Text Labels to Buttons and Elements to Allow Screen Reader Usage

### ISSUE THREAD

**Issue by frastlin** [Cherdyakov/sonification-workstation#93]
Hello,
Currently when I tab through this interface with [NVDA](https://www.nvaccess.org/) I hear "button" "slider", so there is no information about what each slider does. I have not used QT, but there should be a "label" or "title" attribute of each widget class that will add text to the buttons.
Once that is done, I'm pretty sure most of the interface will be usable to screen reader users.
If you made custom widgets, [here is QT documentation](https://doc.qt.io/qt-5/accessible.html) on custom widgets.

**Comment by Cherdyakov** [comment]
This is an excellent suggestion, added to the next sprint.
